packages/carbon-addons-boomerang-react/src/components/FeatureSideNav/FeatureSideNav.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { settings } from 'carbon-components';
+
+FeatureSideNav.propTypes = {
+  header: PropTypes.func,
+  leftBorder: PropTypes.bool,
+  small: PropTypes.bool,
+  children: PropTypes.any,
+  className: PropTypes.string,
+};
+
+const { prefix } = settings;
+
+export function FeatureSideNav(props) {
+  const { header, leftBorder, small, children, className, listClassName } = props;
+  return (
+    <div className={cx(`${prefix}--bmrg-feature-sidenav-container`, className,  {"--left-border": leftBorder, "--small": small })}>
+      {header && header()}
+      <section className={cx(`${prefix}--bmrg-feature-sidenav-links`, listClassName)}>
```

Comment:
Perhaps we should have another simple `FeautureSideNavLinkList` component that has this classname associated with it? If we do that we can just render `children` in this component and make it as flexible as possible. Aka what if in the future we want to pass a footer to the sidenav that should be separate from the sidenav links section?

Reply:
Don't think it needs to have its own top level component that has a story and everything, but would need to be exported.

Reply:
Ok, so to make the `FeautureSideNavLinkList`  we would just create a new component with the `cx(`${prefix}--bmrg-feature-sidenav-links`, listClassName)` wrapping the children and replacing the height 100% by flex-grow?Also we could make other components that would make the sidenav montage easier. Like you said, could be a  `FeautureSideNavFooter` with some default height and self aligned at the bottom with some other configures like the button style (Refference to Create Component SideInfo). Would be pretty much the same thing with a possible FeatureSidenavHeader, but with some default padding, self aligned at the top and maybe a default font-size. since we have many designs for this, there are not much things we can implement by default. What do you think?

Reply:
yeah I think that would work for the `FeautureSideNavLinkList` 

I think a footer could be useful that is just aligned to the bottom. Not sure anything else would be needed for that style wise. Maybe just a 1rem of padding around it. Same thing for the header, might just need a 1rem of padding and thats it. You are right that its hard to build something that is going to provide much value and be flexible enough for whatever might be needed.

If we do those components then would we just render everything as children in the sidenav?

Reply:
Yep, then we won't need the header props anymore, I think that seems better like this.
I pushed an update with these 3 new components.